### DISCUSSION TITLE
Alert Notification

### DISCUSSION THREAD

**Discussion by Paolo-lab** [iTwin/iTwinUI#1691] [Q&A] [answered]
Hi everyone

I am using iTwinUI-layouts for my app, in my HeaderPage I have the SvgNotification notification icon, I would like to make sure, if possible, to add a small red dot or a movement to the Notification icon, when I read external data there are some action that the user must take, I saw in the forum that you discussed it, but I couldn't find the right path

thank you for your precious help

**Comment by mayank99** [comment]
I believe you're talking about the `NotificationMarker` component. See examples: https://itwin.github.io/iTwinUI/react/?story=core--notificationmarker--basic

There is also a `pulsing` animation, but be careful with that. It is for urgent needs, and should not be used in the vast majority of cases.

**Comment by Paolo-lab** [comment]
hi thanks for the reply @mayank99,
I had to update "@itwin/itwinui-react" from version "^1.48.2" to version "^2.12.16", as the component you recommended was not present in my version.
this clearly brings with it a series of problems from the theme to other components, which is why I am preparing to modify the components.
what I can't resolve, however, is that when opening a modal window (before the update it worked correctly)
it gives me an error "ResizeObserver loop completed with undelivered notifications." , without giving me the chance to understand where it is.
I attach an image of the error.
<img width="435" alt="resizeObserver" src="https://github.com/iTwin/iTwinUI/assets/57446652/57a07e17-704c-4b6f-9c39-33017f18935a">

Thanks for your help

**Comment by Paolo-lab** [comment, marked as answer]
hi guys, I solved it, it was a sort inside a React.useEffect that was causing problems

Thank you ..
